src/OmniSharp.MSBuild/ProjectManager.cs

```diff
@@ -93,7 +137,6 @@ protected override void DisposeCore(bool disposing)
 
         public void QueueProjectUpdate(string projectFilePath, bool allowAutoRestore)
         {
-            _logger.LogInformation($"Queue project update for '{projectFilePath}'");
```

Comment:
>_logger.LogInformation($"Queue project update for '{projectFilePath}'"); [](start = 12, length = 72)

Why remove this?

Reply:
accidental change - will restore

---
In reply to: [229800396](https://github.com/OmniSharp/omnisharp-roslyn/pull/1322#discussion_r229800396) [](ancestors = 229800396)